packages/firestore/src/exp/database.ts

```diff
@@ -15,40 +15,41 @@
  * limitations under the License.
  */
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Is it worth separating this group of imports from the rest and adding a comment about why this is necessary?

Reply:
We are now using eslint to sort our imports (possible since we no longer have cycled dependencies). With that, we lost all control over formatting and empty lines.